pulp_ansible/app/urls.py

```diff
@@ -244,6 +245,11 @@
         views_v3.ClientConfigurationView.as_view(),
         name="client-configuration-viewset",
     ),
+    path(
+        "search/collection-versions/",
+        viewsets_v3.CollectionVersionSearchViewSet.as_view({"get": "list", "put": "update"}),
```

Comment:
Do we want the rebuild index action to be a `PUT` request or a `POST` request? I'm more partial to `POST` since we are not updating just one individual index object.